src/twisted/web/_newclient.py

```diff
@@ -279,7 +279,14 @@ def lineReceived(self, line):
                     header = b''.join(self._partialHeader)
                     name, value = header.split(b':', 1)
                     value = value.strip()
-                    self.headerReceived(name, value)
+                    # Handle comma-separated list as field-value list
```

Comment:
I think this violates https://tools.ietf.org/html/rfc7230#section-3.2.6 (it doesn't check for quoting, for example). I also don't think we should be doing this here, since it could break things that do the , parsing later, for example.